content/en/references/external-ports.md

```diff
@@ -3,18 +3,38 @@ title: "External Service Port Range"
 weight: 50
 description: >
     The range of ports used by services not directly provided by LocalStack
-aliases:
-  - /localstack/external-ports/
 ---
 
-Services like [OpenSearch]({{< ref "opensearch" >}}) or [Elasticsearch]({{< ref "elasticsearch" >}}) use external software which binds to separate ports.
+## Introduction
 
-Depending on the configuration of the individual LocalStack service, these services can either be accessed by using the proxy functionality of LocalStack which assigns local domains to these external services.
-For example, if OpenSearch is configured to use the [`OPENSEARCH_ENDPOINT_STRATEGY=domain`]({{< ref "opensearch#endpoints" >}}), a cluster can be reached by using the domain name `<domain-name>.<region>.<engine-type>.localhost.localstack.cloud`.
-Messages coming to those domains are then relayed to the servers running on ports which do not have to be accessible from outside the Docker container.
+LocalStack provides local cloud services, such as [OpenSearch]({{< ref "opensearch" >}}) or [Elasticsearch]({{< ref "elasticsearch" >}}), which utilize external software bound to specific ports. This documentation discusses two approaches to access these external services within LocalStack and explores the concept of an _external service port range_.
```

Comment:
nit: It might be worth mentioning that for opensearch or elasticsearch it's configurable, but there also are services where the domain based addressing is not applicable (f.e. RDS). These other services always us the external service ports.